src/Chapter1.hs

```diff
@@ -640,8 +651,11 @@ Implement a function that returns the first digit of a given number.
 You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
-
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n =
+    if n > 9
+        then firstDigit (div n 10)
+        else n
```

Comment:
Yeah, that is the right idea! 👏🏼 

I see, that for a negative number like -19, this implementation will return the negative number itself instead of the last digit because the first check n <= 9 will always succeed for negative numbers.